### ISSUE TITLE
Does this project replace ExternalDNS ?

### ISSUE THREAD

**Issue by dberardo-com** [k8gb-io/k8gb#1036]
can k8gb work on clusters that already implement ExternalDNS or will there be any conflict?

i see that externalDNS is included in the helm chart, thus i wonder: https://github.com/k8gb-io/k8gb/blob/master/chart/k8gb/values.yaml

**Comment by k0da** [comment]
No, shipped external-dns works only on certain label filter with CRD set as a source.

**Comment by dberardo-com** [comment]
great, thanks! i have also posted some questions on HA concerns in this issue: https://github.com/k8gb-io/k8gb/issues/1035

Perhaps I am missing any more comprehensive online documentation on the tool. at the moment i had a look at the website only, not sure if there is more. 

thanks
